data/json/items/battery.json

```diff
@@ -157,11 +157,11 @@
     "symbol": "=",
     "color": "yellow",
     "ammo_type": [ "battery" ],
-    "count": 100,
-    "capacity": 100,
+    "count": 10,
+    "capacity": 10,
```

Comment:
At an energy density of 0.13 MJ/kg, shouldn't a 10 kJ zinc-carbon cell weigh 77 grams instead of the current 20 g? They're much less dense than lithium chemistries.

Reply:
Where'd you get zinc-carbon? I was guessing this was a random coin cell, which could be a lot of different chemistries.

Reply:
It's more evident when you look at the ingredients in the crafting recipe for making a `light_dry_cell`. Zinc-carbon batteries were added in #54287